src/sparseml/transformers/utils/helpers.py

```diff
@@ -92,6 +96,7 @@ class TaskNames(Enum):
     "special_tokens_map.json",
     "tokenizer_config.json",
 }
+RELEVANT_HF_SUFFIXES = ["json", "md", "bin", "safetensors", "yaml", "yml"]
```

Comment:
Actually we want to have py files as well if there is custom code to run

Reply:
Added! much thanks for the sharp eye!